modules/orbidderBidAdapter.js

```diff
@@ -93,6 +93,9 @@ export const spec = {
       if (bidderRequest && bidderRequest.refererInfo) {
         referer = bidderRequest.refererInfo.page || '';
       }
+      if (bidRequest?.mediaTypes?.video) {
+        delete bidderRequest.mediaTypes.video;
```

Comment:
does this actually work? reading the code I don't think it has any effect on the outgoing request. (also, this `bidderRequest` object is not supposed to have a `mediaType` property).

It would help to have a reproducible issue, and translate that into a unit test.